mappings/net/minecraft/block/entity/EndGatewayBlockEntity.mapping

```diff
@@ -42,17 +43,23 @@ CLASS net/minecraft/class_2643 net/minecraft/block/entity/EndGatewayBlockEntity
 	METHOD method_11421 needsCooldownBeforeTeleporting ()Z
 	METHOD method_30276 canTeleport (Lnet/minecraft/class_1297;)Z
 		ARG 0 entity
-	METHOD method_31698 (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
+	METHOD method_31698 isChunkEmptyAt (Lnet/minecraft/class_3218;Lnet/minecraft/class_243;)Z
```

Comment:
I would name this `isChunkEmpty` per https://github.com/FabricMC/yarn/blob/21w08b/CONVENTIONS.md#conciseness